### ISSUE TITLE
Exeption in watch_balance() for okex exchange - AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by plusInfinity** [ccxt/ccxt#7748]
- OS: Gentoo Linux
- Programming Language version: Python 3.7.7 from Intel's anaconda distribution
- CCXT version: ccxt-1.35.81 ccxtpro-0.4.4

I am using jupyter-notebook to quickly test different parts of ccxt/ccxtpro and got AttributeError for watch_balance() method. In the code below if I change exchange_id to "binance" then everything seems to be working. Replacing watch_balance() with fetch_balance or watch_trades(some_symbol) in the example below works with exchange_id = "okex". I provide password for okex in my credentials as required by okex API docs.

```
import ccxtpro
import json

exchange_id = "okex"

with open("credentials.json", "r") as f:
    credentials = json.load(f)[exchange_id]

exchange = getattr(ccxtpro, exchange_id)({**credentials})
exchange.verbose = True
print(await exchange.watch_balance())
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-9d623aad976a> in async-def-wrapper()

~/.anaconda3/lib/python3.7/site-packages/ccxtpro/okex.py in watch_balance(self, params)
    346         # query = self.omit(params, 'type')
    347         future = self.authenticate()
--> 348         return await self.after_async(future, self.subscribe_to_user_account, params)
    349 
    350     async def subscribe_to_user_account(self, negotiation, params={}):

~/.anaconda3/lib/python3.7/site-packages/ccxtpro/base/exchange.py in after_async(self, future, method, *args)
     82 
     83     async def after_async(self, future, method, *args):
---> 84         return await method(await future, *args)
     85 
     86     async def after_dropped(self, future, method, *args):

~/.anaconda3/lib/python3.7/site-packages/ccxtpro/okex.py in authenticate(self, params)
    333                     self.password,
    334                     timestamp,
--> 335                     self.decode(signature),
    336                 ],
    337             }

~/.anaconda3/lib/python3.7/site-packages/ccxt/base/exchange.py in decode(string)
   1221     @staticmethod
   1222     def decode(string):
-> 1223         return string.decode('latin-1')
   1224 
   1225     @staticmethod

AttributeError: 'str' object has no attribute 'decode'
```

**Comment by kroitor** [comment]
@plusInfinity thx for reporting, will fix it shortly!

**Comment by kroitor** [comment]
Fixed in 0.4.6+, thx again for your feedback! Let us know if you have any further difficulties with it after upgrading to the most recent version of CCXT Pro.

**Comment by plusInfinity** [comment]
Thanks, everything works now with the most recent version of ccxt.pro.
